GettingStarted.md

```diff
@@ -1,46 +1,65 @@
-# Getting started with the commercetools Postman Collection.
+# Getting started
 
-## What are Postman Collections?
+This document provides instructions for installing Postman, for importing commercetools Postman collections and for setting up an environment to get you started.
 
-Postman is an API platform that helps you quickly use REST APIs without requiring any development work. Postman Collections are a group of saved API requests that you can import into the Postman app.
+> **:warning: Note:**
+>
+> - Be aware that Postman automatically synchronizes environment variables (including your API client credentials) to your workspace if logged in.
+> - Use these collections only for development purposes and non-production projects.
 
-## commercetools API collections
+## Disclaimer
 
-* [Platform API](api/)
-* [Import](import/)
-* [Machine Learning](ml/)
+This is not the official commercetools documentation.
+To learn more about commercetools APIs and to find the complete and approved documentation, visit [docs.commercetools.com](http://docs.commercetools.com/).
 
-## Install Postman App
+## Install Postman
 
-Instructions for installing the Postman app or using the web version can be found at https://www.postman.com/downloads/
+Instructions for installing the Postman app or using the web version can be found at [Installing and Updating](https://learning.postman.com/docs/getting-started/installation-and-updates/) in the Postman Learning Center.
 
-## Setting Up Enviornment
-Every root folder in [Platform API](api/), [Import](import/)
-, [Machine Learning](ml/) contains a template.json file. Import this file from the Environments tab.
+## Set up an environment via commercetools API client
 
-![image](https://user-images.githubusercontent.com/4946943/141699003-e989317e-41dc-42c9-b682-eb97f6c8fe6d.png)
+An environment is a set of variables that you can use in your Postman queries. It can be generated and downloaded from the Merchant Center using the API Client to match your project:
 
-This will create an empty variable set environment. 
-![image](https://user-images.githubusercontent.com/4946943/141699543-9f626cd3-5dcf-4b8d-94ad-f0045fc15b44.png)
+1. Create an API client following the instructions provided in [Create an API Client](https://docs.commercetools.com/merchant-center/api-clients#create-an-api-client) in the Merchant Center documentation.
+1. After clicking **Create API Client**, scroll down to the credential downloads section.
```

Comment:
I wrote it this way because it is described until **Create API Client** in the linked guide and therefore already happened - if that makes sense. 
I usually also wouldn't use after in the beginning of a step, but in this case it just made sense. Maybe we can discuss that in the daily sync :)

Reply:
-> scroll down to the bottom.